container/lxd/lxd.go

```diff
@@ -250,6 +269,34 @@ func (manager *containerManager) getImageSources() ([]lxdclient.Remote, error) {
 	return []lxdclient.Remote{remote, lxdclient.CloudImagesRemote, lxdclient.CloudImagesDailyRemote}, nil
 }
 
+// ensureIPv4 retrieves the network for the input name and checks its IPv4
+// configuration. If none is detected, it is set to "auto".
+// The boolean return indicates if modification was necessary.
+func (manager *containerManager) ensureIPv4(netName string) (bool, error) {
```

Comment:
I'm missing where ensureIPv4 is unit tested.

Reply:
As much as it irks me, I've pulled the method off _containerManager_ and used a shim for the client. This now has tests.